src/core/api/harborapi_test.go

```diff
@@ -206,6 +206,13 @@ func init() {
 	beego.Router("/api/internal/switchquota", &InternalAPI{}, "put:SwitchQuota")
 	beego.Router("/api/internal/syncquota", &InternalAPI{}, "post:SyncQuota")
 
+	// Add routes for plugin scanner management
+	scannerAPI := &ScannerAPI{}
+	beego.Router("/api/scanners", scannerAPI, "post:Create;get:List")
```

Comment:
conventionally such resource should better be put under `/api/system/` domain as it's system setting?

Reply:
I suggest using explicit API paths, not adding too many context fragments in the path. `system` is a context subpath, not related to the resources themselves.

Reply:
agree `/api/scanners`

Reply:
OK since it's a 2:1, I'm moving on with disagreement.